src/components/navBar/navBar.module.scss

```diff
@@ -39,38 +39,45 @@ $theme-active: #87d870;
     .logo {
       padding: ($offset + $thickness)+7;
     }
+  }
+  .signInLink {
+    border-radius: 6px;
+    padding: 5px;
+    background-color: $theme-blue;
+    color: $white;
+    border: 1px solid $white;
+    display: flex;
+    justify-content: space-around;
+    font-size: 13px;
```

Comment:
Why are we not using rem here

Reply:
You can reach out to AO of the site for the above decision 😁
Well, you can the pros and cons going ahead with rem

Reply:
I am the AO and I was thinking of probably creating a ticket for it rather.

Reply:
done